mindjunkies/forums/views.py

```
@@ -99,3 +105,24 @@
+            # User is un-reacting
+            topic.reaction.remove(request.user)
+            # Update like_count
+            # topic.like_count = max(0, topic.like_count - 1)  # Ensure it doesn't go below 0
+            topic.save()
+        else:
+            # User is reacting
+            topic.reaction.add(request.user)
+            # Update like_count
+            # topic.like_count += 1
+            topic.save()
+            
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)